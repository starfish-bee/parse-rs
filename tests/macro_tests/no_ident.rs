use parser::*;

#[derive(Operator, Debug, Clone, Copy)]
enum Op {
    #[assoc = "right"]
    #[ident = "#"]
    Hash,
    #[assoc = "right"]
    #[ident = "^"]
    Caret,
    QMark,
}

fn main() {}
