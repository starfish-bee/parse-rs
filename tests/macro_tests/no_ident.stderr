error: expected list attribute 'ident('x')'
  --> $DIR/no_ident.rs:11:5
   |
11 |     QMark,
   |     ^^^^^
